### ISSUE TITLE
Wrong number of arguments for `prose.lint_files`

### ISSUE THREAD

**Issue by BasThomas** [dbgrandi/danger-proselint#20]
This was working fine before, but seems to be failing for some reason at this point over at [Swift Weekly Brief](https://github.com/SwiftWeekly/swiftweekly.github.io/pull/281#issuecomment-310440414). Any idea as to what causes this?

Dangerfile: https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/Dangerfile
Gemfile: https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/Gemfile
Gemfile.lock: https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/Gemfile.lock
Danger versions:
```
danger (5.3.2)
      claide (~> 1.0)
      claide-plugins (>= 0.9.2)
      colored2 (~> 3.1)
      cork (~> 0.1)
      faraday (~> 0.9)
      faraday-http-cache (~> 1.0)
      git (~> 1)
      kramdown (~> 1.5)
      octokit (~> 4.2)
      terminal-table (~> 1)
    danger-prose (2.0.4)
      danger
```

**Comment by orta** [comment]
Looks like dying inside this lib:

```
	from /home/travis/build/SwiftWeekly/swiftweekly.github.io/vendor/bundle/ruby/2.1.0/gems/danger-prose-2.0.4/lib/danger_plugin.rb:62:in `[]'
	from /home/travis/build/SwiftWeekly/swiftweekly.github.io/vendor/bundle/ruby/2.1.0/gems/danger-prose-2.0.4/lib/danger_plugin.rb:62:in `block in lint_files'
	from /home/travis/build/SwiftWeekly/swiftweekly.github.io/vendor/bundle/ruby/2.1.0/gems/danger-prose-2.0.4/lib/danger_plugin.rb:179:in `with_proselint_disabled'
	from /home/travis/build/SwiftWeekly/swiftweekly.github.io/vendor/bundle/ruby/2.1.0/gems/danger-prose-2.0.4/lib/danger_plugin.rb:60:in `lint_files'
```

**Comment by dbgrandi** [comment]
Looking into it...

**Comment by dbgrandi** [comment]
@BasThomas Should be fixed with https://github.com/dbgrandi/danger-prose/pull/23 and I published this as danger-prose 2.0.5

**Comment by dbgrandi** [comment]
BTW, thanks for reporting this and sorry you got hit with this. Big fan of SwiftWeekly. :)
